### PR TITLE
Feature: Tautulli and Emby/Jellyfin User Name option

### PR DESCRIPTION
**Pull request by brikim**
## Proposed change

Added a new optional flag that can be added to the widget enableUser. If true the username will be added in parenthesis behind the title. Updated the documentation stating that this is an optional field and defaults to false.

TautulliEmbyJellyfinUsername
TautulliJellyfinEnableUserConfig

Closes #1233  #151 

## Type of change

- [ ] New service widget
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Documentation only
- [ ] Other (please explain)

## Checklist:

- [x] If applicable, I have added corresponding documentation changes.
- [x] If applicable, I have reviewed the [feature](https://gethomepage.dev/latest/more/development/#new-feature-guidelines) and / or [service widget guidelines](https://gethomepage.dev/latest/more/development/#service-widget-guidelines).
- [x] I have checked that all code style checks pass using [pre-commit hooks](https://gethomepage.dev/latest/more/development/#code-formatting-with-pre-commit-hooks) and [linting checks](https://gethomepage.dev/latest/more/development/#code-linting).
- [x] If applicable, I have tested my code for new features & regressions on both mobile & desktop devices, using the latest version of major browsers.

### CODE REVIEW

File: .github/dependabot.yml
Comment (brikim) [inline .github/dependabot.yml:1]: Not sure why this file is showing up I don't see it as a change on my branch

**Comment by shamoon** [comment]
You need to rebase this PR

File: .github/dependabot.yml
Comment (brikim) [reply .github/dependabot.yml:1]: Yeah I tried that and I'm confused cuz it says I'm based off of the latest main branch

Comment (brikim) [reply .github/dependabot.yml:1]: Yeah that was the first thing I tried to do was a rebase. I'm not sure if it matters but I updated my fork and then pulled the latest main branch and made my changes directly on that.

**Comment by shamoon** [comment]
Please recreate with a clean history

**Pull request closed without merging**